bigc/api_client.py

```diff
@@ -80,11 +87,11 @@ def perform_request() -> Any:
             try:
                 return perform_request()
             except (
-                InternalServerError,
-                BadGatewayError,
-                ServiceUnavailableError,
-                GatewayTimeoutError,
-                BigCommerceNetworkError,
+                    InternalServerError,
+                    BadGatewayError,
+                    ServiceUnavailableError,
+                    GatewayTimeoutError,
+                    BigCommerceNetworkError,
```

Comment:
I thought the extra indent was to visually separate params from the function body. this feels different?